smithy-templates.json

```
@@ -0,0 +1,125 @@
+{
+  "name": "smithy-gradle-plugin",
+  "templates": {
+    "include-in-sourceset": {
```

Should this be plural?
```suggestion
    "includes-in-sourceset": {
```